Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,55 @@
+# Microcontroller Signoff
+## FUNCTION OF THE SUBSYSTEM
+The microcontroller and its code are the centerpieces of the entire device.  They are responsible for operating, managing, communicating with, maintaining, and collaborating with all of the other subsystems and acts as both the brain and body of the device.  All the data and instructions must flow through the microcontroller at some point in its lifetime.  This gives a common place for all of the data to go and be readily available for whichever subsystem has need of it.  The hardware will be responsible for the actual actions and completion of any given task.  The code will be responsible for controlling the hardware and manipulating data.  This subsystem will be where most of the coding for the project is done as the operating code will be the largest by far.  The code itself will be where the “magic” happens as it allows for interfacing with the hardware and makes it more simple to utilize.
+
+These jobs require a sufficiently powerful board.  One that has ample storage, plenty of physical space, decent processing power, sufficient support, and none restrictive.  Before choosing components and detailing the design, the specific constraints of the subsystem need to be known and laid out.  Then any potential solutions can be compared to the constraints to test viability and fulfillment of the subsystem’s purpose. 
+
+## CONSTRAINTS ON THE SUBSYSTEM
+The subsystem constraints are placed to ensure that it can fulfill its job in a satisfactory manner.  The constraints given below are generalized to allow for comparison of any potential contender for the microcontroller.  Constraints on the subsystem are as follows:
```

Comment:
Constraints should be specific and are best presented as a list. Where appropriate, tie the constraint back to a shall statement to provide justification for that constraint. To make it easy to read write the constraint first, then the justification.